labs/src/main/scala/org/scalalabs/basic/lab03/PatternMatchingExercise.scala

```diff
@@ -23,7 +23,12 @@ object PatternMatchingExercise {
    *************************************************************************/
 
   def describeLanguage(s: String): String = {
-    error("fix me")
+    // Not pattern matching, just for fun.
+    Map("Java" -> "OOP", "Smalltalk" -> "OOP",
+        "Clojure" -> "Functional", "Haskell" -> "Functional",
+        "Scala" -> "Hybrid",
+        "C" -> "Procedural",
+        "Oz" -> "Unknown")(s)
   }
```

Comment:
might have been more inuitive to just match s and use each of your mappings as a case